### PR TITLE
Frontend_Web_Development_HTML_CSS_JS: The DOM Examples

### LINKED ISSUE #1879: Frontend_Web_Development_HTML_CSS_JS: The DOM Examples
### Description

5 examples of The DOM javascript using HTML, CSS & Js.

### Domain

Frontend Dev HTML/CSS/JS

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by amankumar11**
<hr>

## Description 📜

Added The DOM js examples in Web_Development/Javascript/7.The DOM/DOM - Examples/ in branch Frontend_Web_Development_HTML_CSS_JS.
.
Examples include 
1. Background Color Changer
2. Counter
3. Tab Switching
4. Modal
5. Sort courses
Fixes #1879 
@code-reaper08 pls check if the changes are up to the limit.

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Frontend Dev HTML/CSS/JS

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸
Yes, the readme of each example has a gif of the example attached.

<hr>

### CODE REVIEW

File: Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Example1 - Background color changer
+
+<img src="example1.gif"/>
```
Comment (praveenscience) [inline Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/README.md:3]: Use Code blocks.

File: Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Example1 - Background color changer
```
Comment (praveenscience) [inline Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/README.md:1]: Use normal English. Give proper grammar. Space after Example.

File: Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/changeColor.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+function getRandom1(max){
+    return Math.floor(Math.random() * max);
+}
+function getRandom2(max){
+    return Math.floor(Math.random() * max);
+}
+function getRandom3(max){
+    return Math.floor(Math.random() * max);
+}
```
Comment (praveenscience) [inline Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/changeColor.js:24]: Let's keep to markdown files not code files.

File: Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/randomColor.css [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    color: red;
+}
+
+.github{
+    text-decoration: none;
+}
+
+.gwoc{
+    color: tomato;
+}
```
Comment (praveenscience) [inline Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/randomColor.css:66]: Let's keep to markdown files not code files.
Make sure the code is indented properly.

File: Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/randomColor.html [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+            <p>Background Color: <span class="color">RGB(255,255,255)</span></p>
+        </div>
+        <button class="changebtn" id="colorbtn">Change!</button>
+    </div>
+    <div class="footer">
+        <p>Made with <span class="heart">&hearts;</span> by <span><a class="github" href="https://github.com/amankumar11">Aman Kumar</a></span> for <span class="gwoc">GWOC'21</span></p>
+    </div>
+    <script src="./changeColor.js"></script>
+</body>
+</html>
```
Comment (praveenscience) [inline Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/randomColor.html:22]: Let's keep to markdown files not code files.
Make sure the code is indented properly.

File: Web_Development/Javascript/7.The DOM/DOM - Examples/Example3 - Tab switching/README.md [hunk 6]
```diff
@@ -0,0 +1,3 @@
+# Example3 - Tab Switching
+
+<img src='example3.gif'>
```
Comment (praveenscience) [inline Web_Development/Javascript/7.The DOM/DOM - Examples/Example3 - Tab switching/README.md:3]: Use Code blocks.

File: Web_Development/Javascript/7.The DOM/DOM - Examples/README.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
+# The DOM Javascript Examples
+
+## 1. Background Color Changer
+## 2. Counter
+## 3. Tab Switching
+## 4. Modal
+## 5. Sort courses
+<br>
+Contributor: <a href="https://github.com/amankumar11">Aman Kumar</a>
+
```
Comment (praveenscience) [inline Web_Development/Javascript/7.The DOM/DOM - Examples/README.md:10]: No need of this.

**Review by praveenscience — CHANGES_REQUESTED** [review]
Please check the review comments and let me know. 😊

File: Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/README.md (hunk 1 above)
Comment (amankumar11) [reply Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/README.md:3]: Required changes made in the latest commit

File: Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/README.md (hunk 2 above)
Comment (amankumar11) [reply Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/README.md]: Required changes made in latest commit.

File: Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/changeColor.js (hunk 3 above)
Comment (amankumar11) [reply Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/changeColor.js]: Required changes made in latest commit.

File: Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/randomColor.css (hunk 4 above)
Comment (amankumar11) [reply Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/randomColor.css]: Required changes made in latest commit.

File: Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/randomColor.html (hunk 5 above)
Comment (amankumar11) [reply Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/randomColor.html]: Required changes made in the latest commit.

File: Web_Development/Javascript/7.The DOM/DOM - Examples/Example3 - Tab switching/README.md (hunk 6 above)
Comment (amankumar11) [reply Web_Development/Javascript/7.The DOM/DOM - Examples/Example3 - Tab switching/README.md]: Required changes made in the latest commit.

File: Web_Development/Javascript/7.The DOM/DOM - Examples/README.md (hunk 7 above)
Comment (amankumar11) [reply Web_Development/Javascript/7.The DOM/DOM - Examples/README.md]: Required changes made in the latest commit.

**Comment by amankumar11** [comment]
Required changes have been made.
Pls review @praveenscience

File: Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/README.md (hunk 1 above)
Comment (praveenscience) [reply Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/README.md:3]: Still looks the same?

File: Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/README.md [hunk 8]
```diff
@@ -0,0 +1,130 @@
+# Example 1 - Background color changer
+
+<img src="example1.gif"/>
+<br>
+<br>
```
Comment (praveenscience) [inline Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/README.md:5]: No need of HTML inside Markdown please.

**Review by praveenscience — CHANGES_REQUESTED** [review]
Convert all the HTML into Markdown please. We should not have any HTML code inside MD files.

File: Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/README.md (hunk 8 above)
Comment (amankumar11) [reply Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/README.md]: Html code is deleted

**Comment by amankumar11** [comment]
Please review the latest commit, the required changes are made

**Comment by Gauravmauryaios** [comment]
I want to work on making a complete documentation on the topic Conventions and Best Practices.
Please /assign this to me.

**Comment by amankumar11** [comment]
@praveenscience pls merge the changes it's been more than 7 days since the PR is opened.

**Pull request merged**